sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/models/student-scholastic-standings.dto.ts

```diff
@@ -31,11 +31,17 @@ export class ScholasticStandingAPIInDTO {
   data: ScholasticStandingData;
 }
 
+/** Represents the scholastic standing submitted details. */
```

Comment:
Minor. Can we have the comments in this format.?

![image](https://github.com/bcgov/SIMS/assets/54600590/b01a2989-1d57-40fa-93e9-fcd2d3211591)

Same for the next as well.